README.md

```diff
@@ -1,61 +1,86 @@
+# Thoth Tutorial - manage your dependencies in Jupyter notebooks.
+
+This tutorial is used to show how to manage dependencies for Jupyter Notebooks to allow reproducibility and shareability. You will learn how to move from local to cloud development using [Operate First][1] environment and how you can enable contributions on your project.
+
+Once the tutorial is completed, you will be able to run your work on [Project Meteor][2].
+
+## Environment and Tools
+
+In this tutorial the following technologies are going to be used:
+
+- [jupyterlab-requirements][3] library. ([Project Thoth][4], JupyterLab extension for dependency management)
+- [JupyterHub][5] to launch images with Jupyter tooling.
+
+If you use the [Operate First][1] environment, the above requirements will be already setup for you selecting `elyra-image` from [JupyterHub][5].
+
+### Operate First Open Environment
+
+[Operate First][1] is an open infrastructure environment started at Red Hat's Office of the CTO. It has been selected to run this tutorial since it is an open source initiative that fulfills all the requirements stated above. Anyone with a Google account can log in and start developing. To learn more about Operate First, visit the [website](https://www.operate-first.cloud/) or [GitHub community](https://github.com/operate-first).
+
+
+## Project templates
+
+The tutorial structure follows this [project template][6]. Having structure in a project ensures all the pieces required for the project lifecycle are present and easily discoverable.
+
+If you want to use this template for your AI project, go to this [project template][6] and click the `Use the template` button provided in the repository for your new projects.
+
+<div style="text-align:center">
+<img alt="AI Project Template" src="https://raw.githubusercontent.com/AICoE/manage-dependencies-tutorial/master/docs/images/AIProjectTemplate.png">
+</div>
+
+
+## Automated pipelines and bots for your GitHub project
+
+- [Kebechet Bot][7] to keep your dependencies fresh and up to date receiving recommendations and justifications using AI.
+
+- [AICoE CI][8] to support your AI project lifecycle.
+
+All these tools are integrated with the [project template][6], so most additions are already set for you.
+These bots and pipelines exist to automate many of the manual GitOps tasks. For example, in order to deploy your application, you may need to create a container image. GitHub templates integrated with bots can provide you with automated pipelines triggered depending on what you need (e.g. release (patch, minor, major), deliver a container image, or update your dependencies).
+
+
+## GitOps for reproducibility, portability, traceability with AI support
+
+Nowadays, developers (including data scientists) use Git and GitOps practices to store and share code on development platforms such as GitHub. GitOps best practices allow for reproducibility and traceability in projects.
```

Comment:
This is all very interesting, but I think it is a bit unclear to the newcomer how does this apply to the topic of this tutorial: what is the relationship of GitOps with dependency management? do I need a project template so that I can manage my dependencies?

Maybe have a section called "Context" or "Background" with all these topics and references?

Reply:
Thanks! We can definetly expand and separate that part in background section!!